pkgs/stdenv/generic/setup.sh

```diff
@@ -1323,6 +1323,15 @@ checkPhase() {
 installPhase() {
     runHook preInstall
 
+    # Dont reuse 'foundMakefile' set in buildPhase, a makefile may have been created in buildPhase
+    if [[ -z "${makeFlags-}" && -z "${makefile:-}" && ! ( -e Makefile || -e makefile || -e GNUmakefile ) ]]; then
+        echo "no Makefile or custom installPhase, doing nothing"
+        runHook postInstall
+        return
+    else
+        foundMakefile=1
+    fi
+
```

Comment:
Shouldn't this be moved after the three-line `if..mkdir..fi` "paragraph" that follows it?

Otherwise that "paragraph" is skipped when there is no `Makefile`.  I believe the intent is simply to skip the `make install`, right?

Reply:
```
    if [ -n "$prefix" ]; then
        mkdir -p "$prefix"
    fi
```
will create a empty $prefix($out) so no

Reply:
Ah, I see.

For some reason I can't mark this comment as "resolved".  Weird.